### PR TITLE
Allow the window option to be overridden as part of the overrides paramater

### PR DESCRIPTION
**Pull request by lbeder**
According to the documentation, it should've been possible to override the window option as well as the burst and the rate options, but it was actually ignored.

I'd appreciate if you could merge this PR, as I desperately need it for my current project.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -62,15 +62,18 @@ Throttle.prototype.rateLimit = function (key, cb) {
 
   var burst = self.burst
   var rate = self.rate
+  var window = self.window
```
Comment (brycebaril) [inline index.js]: Probably should avoid the variable name `window` if just for the fact that most syntax highlighters will treat it as the global `window` object.

**Comment by brycebaril** [comment]
Hi @lbeder -- thanks for catching this! Rename the `window` variable to something else and I'll get this out for you today.

File: index.js (hunk 1 above)
Comment (lbeder) [inline index.js]: I agree. What about 'wnd' or even 'durationWindow' instead?

**Comment by lbeder** [comment]
Done and done! :)

**Comment by brycebaril** [comment]
Released as v1.3.0, thanks!

**Pull request merged**